index.js

```diff
@@ -103,17 +103,18 @@ function run (cmd, cwd, env, cb) {
 
 function build (target, runtime, opts, cb) {
   var args = [
-    'rebuild',
-    '--target=' + target
+    'install',
```

Comment:
Can we consider `npm run install` instead of `npm install`? The latter will slow builds down (doing unnecessary work), especially with `prebuildify-cross`